test/Microsoft.TestTemplates.Acceptance.Tests/DotnetCoreTemplateTests.cs

```diff
@@ -1,24 +1,21 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.TestTemplates.Acceptance.Tests
-{
+namespace Microsoft.TestTemplates.Acceptance.Tests {
```

Comment:
The latest VS preview is breaking curly brace position, you will have to manually fix them all

Reply:
done